content/tr/style-guide/_index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Stil Kılavuzu
+toc_hide: true
+status: Completed
+menu:
+  main:
+    weight: 10
+---
+
+Bu stil kılavuzu, Sözlük hedef kitlesini, tanım yapısını, gerekli ayrıntı düzeyini ve tutarlı bir stili nasıl koruyacağınızı anlamanıza yardımcı olacaktır.
+
+Cloud Native Sözlüğü, CNCF repository’sinin [varsayılan stil kılavuzunu](https://github.com/cncf/foundation/blob/master/style-guide.md) takip eder. Bunlara ek olarak, aşağıdaki kurallara uyar:
+
+1. Teknik jargon ve moda sözcüklerden kaçınarak basit ve erişilebilir bir dil kullanın
+2. [Gündelik konuşma dilinden](https://tr.wikipedia.org/wiki/G%C3%BCnl%C3%BCk_dil) kaçının
+3. Mecazi olmayan, somut bir dil kullanın
+4. Kısaltmalardan kaçının
+5. [Edilgen dili](https://sozluk.gov.tr/?/edilgen+fiil) idareli kullanın
+6. İfadeleri olumlu bir biçimde ifade etmeyi hedefleyin
+7. Alıntılar dışında ünlem işaretini kullanmayın
+8. Abartılı bir dil kullanmayın
+9. Tekrardan kaçının
+10. Kısa ve öz olun
+
+## Hedef Kitle
+
+Sözlük, teknik ve teknik olmayan herkes için yazılmıştır. Lütfen tanımları basit terimlerle açıklayın ve okuyucunun teknik bilgiye sahip olduğunu varsaymayın.
+Konu ile ilgili daha fazla bilgi aşağıda [Tanım](#tanım) bölümünde yer almaktadır.
+
+## Asgari geçerli tanım
+
+Cloud Native terimlerini herkesin anlamasını kolaylaştırmayı amaçlıyoruz.
+Bu sebeple sadeliğe odaklanıyoruz.
+En azından teknik açıdan geçerli *asgari bir tanım* sağlarken, teknolojiyi kullanan herkesin ilişkilendirebileceği örneklerle açık ve basit bir dil kullanın.
+Bağlam ve örneklerden tasarruf etmek istemiyoruz - sonuçta bunlar okuyucunun kavramı anlamasına yardımcı oluyor - ancak, kavramı anlamak için teknik bir ayrıntıyı anlatmak gerekmiyorsa, bu ayrıntıyı es geçebiliriz.
+Burada amaç, işleri olduğundan daha fazla bir karışıklığa getirmemek.
+Okuyucu, temel fikri anladığında başka kaynakları kullanarak konu hakkında daha detaylı bilgi edinebilir.
+Bu kısım, sözlüğün kapsamının dışında kalmaktadır.
+
+## Tanım şablonu
+
+Her sözlük terimi bir markdown dosyasında tutulur ve aşağıdaki şablonu takip eder:
+
+```md
+---
+title: 
+status: 
+category: 
```

Comment:
```suggestion
title:
status:
category:
```

Just noticed `NBSP` chars here on my IDE

Reply:
Must have come from the gdoc2md. Thank you for catching this.